bedrock/firefox/views.py

```diff
@@ -443,7 +443,10 @@ def hello(request):
         'zh-TW': 'https://videos.cdn.mozilla.net/uploads/FirefoxHello/firefoxhello_intro_chinese'
     }
 
-    return l10n_utils.render(request, 'firefox/hello/index.html',
+    if (waffle.switch_is_active('firefox-hello-2016')):
```

Comment:
Not sure what you mean. We could set a calendar appointment. We can't (or more correctly I think shouldn't) schedule waffle switches right now just due to how complex it now is with 3 separate deployments. We can enable the switch whenever we want though.